### ISSUE TITLE
java.lang.IllegalArgumentException: Invalid offsets

### ISSUE THREAD

**Issue by Hipapheralkus** [vulnersCom/burp-vulners-scanner#9]
I can see following output in the Error log for this extender:
```
	at burp.fcg.addScanIssue(Unknown Source)
	at burp.uvb.addScanIssue(Unknown Source)
	at burp.jbf.addScanIssue(Unknown Source)
	at burp.VulnersService$1.onScannerSuccess(VulnersService.java:89)
	at burp.VulnersRestCallback.onSuccess(VulnersRestCallback.java:42)
	at burp.VulnersRestCallback.completed(VulnersRestCallback.java:60)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:82)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:75)
	at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:119)
	at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:177)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:432)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:325)
	at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:267)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:123)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:164)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:339)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:317)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:278)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:106)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:590)
	at java.lang.Thread.run(Thread.java:745)
java.lang.IllegalArgumentException: Invalid offsets: the list should be in sequence and offsets should not overlap.
	at burp.fcg.addScanIssue(Unknown Source)
	at burp.uvb.addScanIssue(Unknown Source)
	at burp.jbf.addScanIssue(Unknown Source)
	at burp.VulnersService$1.onScannerSuccess(VulnersService.java:89)
	at burp.VulnersRestCallback.onSuccess(VulnersRestCallback.java:42)
	at burp.VulnersRestCallback.completed(VulnersRestCallback.java:60)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:82)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:75)
	at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:119)
	at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:177)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:432)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:325)
	at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:267)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:123)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:164)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:339)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:317)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:278)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:106)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:590)
	at java.lang.Thread.run(Thread.java:745)
java.lang.IllegalArgumentException: Invalid offsets: the list should be in sequence and offsets should not overlap.
	at burp.fcg.addScanIssue(Unknown Source)
	at burp.uvb.addScanIssue(Unknown Source)
	at burp.jbf.addScanIssue(Unknown Source)
	at burp.VulnersService$1.onFail(VulnersService.java:103)
	at burp.VulnersRestCallback.completed(VulnersRestCallback.java:56)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:82)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:75)
	at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:119)
	at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:177)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:432)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:325)
	at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:267)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:123)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:164)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:339)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:317)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:278)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:106)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:590)
	at java.lang.Thread.run(Thread.java:745)
java.lang.IllegalArgumentException: Invalid offsets: the list should be in sequence and offsets should not overlap.
	at burp.fcg.addScanIssue(Unknown Source)
	at burp.uvb.addScanIssue(Unknown Source)
	at burp.jbf.addScanIssue(Unknown Source)
	at burp.VulnersService$1.onFail(VulnersService.java:103)
	at burp.VulnersRestCallback.completed(VulnersRestCallback.java:56)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:82)
	at com.mashape.unirest.http.HttpClientHelper$1.completed(HttpClientHelper.java:75)
	at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:119)
	at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:177)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:432)
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:325)
	at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:267)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:123)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:164)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:339)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:317)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:278)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:106)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:590)
	at java.lang.Thread.run(Thread.java:745)

```
Is it possible to fix it? I didn't add/alter any rules, only those default ones, freshly loaded from vulners.com are used.
Thanks

**Comment by vankyver** [comment]
Hi!
Thanks for your help, we fixed problem. Could you please recheck?

**Comment by vankyver** [comment]
Issue finally fixed in 1.2
